index.js

```diff
@@ -26,13 +26,13 @@ var appServer = function(config) {
     app_dir: 'apps'
   };
 
-  self.config = defaults(config, defaultOptions);
+  self.config = utils.defaults(config, defaultOptions);
```

Comment:
I just moved defaults in `utils.js` since it is technically a 'utility' function. I can move it back though.